Add rituals section to brand page. (#5183)
fleetdm/fleet: handbook/brand.md

  * Avoid using SVGs or icon fonts.
3. Click the __Export__ button.

## Brand rituals

The following rituals are engaged in by the  directly responsible individual (DRI) and at the frequency specified for the ritual.

| Ritual                       | Description                                         | DRI               |
|:-----------------------------|:----------------------------------------------------|-------------------|
| Kanban board check  | Check internal kanban boards and ensure tasks are on track and being pushed through to "delivered." Occurs daily. | Mike Thomas  |  
| Documentation quality ritual      | Review pull requests for tone, brand consistenty and quality assurance before merging. Occurs daily. | Mike Thomas, Desmi Dizney |

> | Tweet review  | Review tweets for tone, brand consistency, and quality assurance. Occurs daily.    | Mike Thomas, Desmi Dizney   |

Reviewer: ```suggestion
| Tweet review | Daily | Review tweets for tone and brand consistency | Mike Thomas |
```

> | Slack monitoring     | Check Fleey/osQuery channels on Slack for community questions, ensure questions are responded to and logged. Occurs daily. | Kathy Satterlee, Kelvin Omereshone   |

Reviewer: ```suggestion
| Community Slack  | Daily   | Check Fleet and osquery Slack channels for community questions, ensure questions are responded to and logged | Kathy Satterlee |
```

Reviewer: I rephrased "Slack monitoring". "Monitoring sounds a bit ominous 👁

Reviewer: DRI should be one person (i.e., an "individual".)

> | Social channels check-in     | Check social channels for community questions and ensure they are responded to.  Generate dev advocacy related content. Occurs daily. | Kathy Satterlee, Kelvin Omereshone   |

Reviewer: ```suggestion
| Social media check-in     |  Daily | Check social media for community questions and ensure they are responded to.  Generate dev advocacy-related content | Kelvin Omereshone   |
```

> | Weekly blogpost review | Review blogposts for tone, brand consistency and quality assurance. | Mike Thomas, Desmi Dizney|

Reviewer: ```suggestion
| Weekly article review | Weekly | Review articles for tone and brand consistency. | Mike Thomas |
```

> | Weekly blogpost graphic | Create graphic for the weekly blogpost. | Mike Thomas |

Reviewer: ```suggestion
| Weekly article graphic | Weekly | Create a graphic for the weekly article | Mike Thomas |
```

> | Outside contributor follow up | Bring pull requests from outside contributors to engineering and ensure they are merged promptly and promoted. Occurs weekly. | Kathy Satterlee, Kelvin Omereshone |

Reviewer: ```suggestion
| Outside contributor follow up | Weekly | Bring pull requests from outside contributors to engineering and ensure they are merged promptly and promoted | Kathy Satterlee |
```

> | FAQs documentation update | Turn questions answered from Fleet and osQuery Slack channels into FAQs in Fleet’s documentation. Occurs weekly. | Kathy Satterlee, Kelvin Omereshone |

Reviewer: ```suggestion
| Documentation update | Weekly | Turn questions answered from Fleet and osquery Slack into FAQs in Fleet’s docs | Kathy Satterlee |
```

> | Digital experience planning  | Prioritize and assigns issues to relevant personnel based off of current OKR goals. Occurs every three weeks in concurrence with the Fleet release cycle. | Mike Thomas  |

Reviewer: ```suggestion
| Digital experience planning  | Three weeks | Prioritize and assigns issues to relevant personnel based on current goals and quarterly OKRs | Mike Thomas |
```

> | Browser compatability check  | Check for browser compatability once a month. | Mike Thomas |

Reviewer: ```suggestion
| Browser compatibility check | Monthly | Check browser compatibility for the website | Eric Shaw |
```
